bin/main.ml

```diff
@@ -434,15 +434,21 @@ let rules =
   in
   (term, Term.info "rules" ~doc ~man)
 
-let get_prefix context ~from_command_line =
-  match from_command_line with
-  | Some p -> Fiber.return (Path.of_string p)
-  | None -> Context.install_prefix context
-
-let get_libdir context ~prefix ~libdir_from_command_line =
-  match libdir_from_command_line with
-  | Some p -> Fiber.return (Some (Path.relative prefix p))
-  | None -> Context.install_ocaml_libdir context
+let get_dirs context ~prefix_from_command_line ~libdir_from_command_line =
+  match prefix_from_command_line with
+  | Some p ->
+    let prefix = Path.of_string p in
+    let dir = Option.value ~default:"lib" libdir_from_command_line in
+    Fiber.return (prefix, Some (Path.relative prefix dir))
+  | None ->
+    Context.install_prefix context >>= fun prefix ->
+    let libdir_fiber =
```

Comment:
A bit of a personal preference: but the `_fiber` suffix is redundant and it's obvious from the type

Reply:
That was to avoid a `libdir >>= fun libdir ->` below, but you're right that it reads better